Library/Homebrew/dev-cmd/bump.rb

```diff
@@ -64,9 +68,26 @@ def run
         Homebrew.with_no_api_env do
           eval_all = args.eval_all? || Homebrew::EnvConfig.eval_all?
 
-          formulae_and_casks = if args.tap
+          formulae_and_casks = if args.auto?
+            raise UsageError, "`--formula` or `--cask` must be passed with `--auto`." if !args.formula? && !args.cask?
```

Comment:
Would be nicer to have a `autobump.yml` with `casks` and `formulae` keys in my opinion.

Reply:
@carlocab agreed, not worth blocking this PR on 👍🏻